### ISSUE TITLE
Is there a way to change timezone in Mi home app video record?

### ISSUE THREAD

**Issue by leonidas2522** [EliasKotlyar/Xiaomi-Dafang-Hacks#86]

**Comment by hubmon** [comment]
Have you asked the Manufacturer or Seller?
http://en.miui.com/forum.php?mod=viewthread&tid=1218678&highlight=dafang

**Comment by leonidas2522** [comment]
Yes, they say nothing.
The video record always show GMT+8 (China time) timestamp.
I think only hope to change timezone in Mi home video recordis hack.

**Comment by hubmon** [comment]
Am in USA EST zone, my time stamp display correctly. Set to China, Language=default
Firmware current? 
Tried Forum suggestions?

**Comment by leonidas2522** [comment]
I tried many firmware. Set to China mainland, English language. Ios.
When I set the option "distorsion correction" on, live video timestamp show correctly(mine ICT-7). 
If I set distorsion correction off, live video timestamp show China time(GMT+8). 
But no matter what "distorsion correction" on/off , the record video on sdcard only show China time.

**Comment by hubmon** [comment]
Yep. View has local timezone, Recording has Server(China) timezone.

**Comment by leonidas2522** [comment]
I have 1 Dafang and 2 Xiaofang.
Today I tried Xiaofang hack. The web interface is similar to Dafang hack. 
I set timezone to my local timezone(ICT-7) and then disable hack to use Mi home app continuous recording.
When I connect to Mi home app, I wonder live video timestamp and recording video timestamp on sdcard both show my local timezone(ICT-7).

I tried install Dafang cfw from this Dafang hack and set timezone to my local timezone.
The rtsp steam work great. Rtsp timestamp show my timezone correctly.
But when I removed sdcard to use Mi home app. Live video and record video still show china time.
I tried connect Dafang via FTP and edit TZ file directly in etc folder but not work too. 

I hope if Dafang hack can change timezone im Mi home app, that is a wonderful function.
Does anyone have any other suggestions?

**Comment by madmax2** [comment]
@leonidas2522 where is the distorsion correction setting?

can you post an image of that screen?
Also which version of mi home you to have this setting?
